backend/app/controllers/spree/admin/dashboards_controller.rb

```diff
@@ -3,6 +3,9 @@
 module Spree
   module Admin
     class DashboardsController < BaseController
+      class << self
+         Spree.deprecator.warn "The Dashboards controller is deprecated. Please use the new admin dashboard."
+      end
```

Comment:
I think if we put this in a `before_action` the code would only run if the controller is used during runtime and not during app boot. Could you try?

```rb
before_action do
  Spree.deprecator.warn "The Dashboards controller is deprecated. Please use the new admin dashboard."
end
```

Reply:
Also I find the message a bit confusing. To me it is not clear which one is deprecated and which I should use.

The deprecated controller can simply be logged by using

```rb
"The #{self.class.name} controller" ...
```

in the message. Could you please give a more precise hint which the new controller is? Maybe by using a path or a more precise name?

Reply:
Updated controller.

Reply:
There isn't a new admin dashboard. This dashboards feature was something that was added to the project, but never actually used in core (and I've never seen it used in store either.) We're not removing this in place of something else, we're just removing it.

Reply:
Yes, for now there's no replacement. We can say something like:

"The Dashboards controller is deprecated. If you still use dashboards, please copy all controllers and views from solidus_backend to your application."

Reply:
Also, there's a test failure that I think is related!

Reply:
@kennyadsl Is the above language the same for the DashboardDisplay module, the Home controller and view?

Reply:
Maybe we can use a single message that is more generic and we can apply to all. Up to you to choose it, but let me know if I can help.

Reply:
See latest commit.